backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala

```diff
@@ -0,0 +1,96 @@
+package mark_marker.trainer
+
+import java.io._
+import java.sql.DriverManager
+import java.sql.Connection
+
+import edu.stanford.nlp.classify.{Classifier, ColumnDataClassifier}
+
+import scala.io.Source
+
+class DatabaseConnector {
+  private val driver = "com.mysql.cj.jdbc.Driver"
+  private val url = "jdbc:mysql://tc.jstudios.ovh:3306/Automarker"
+  private val username = "python"
+  private val psw = findPsw("../dbpass.txt")
+  private var connection: Connection = _
+
+  def findPsw(path: String) = {
+    Source.fromFile(new File(path)).mkString
+  }
+
+  def connect() = {
+    Class.forName(driver)
+    DriverManager.getConnection(url, username, psw)
+    connection = DriverManager.getConnection(url, username, psw)
+  }
+
+  def close() = connection.close()
+
+  def getTrainingData(exercise: String): String = {
+    val stat = connection.prepareStatement(s"SELECT training_set FROM MarkMarkerTraining WHERE exercise_name = ?")
+    stat.setString(1, exercise)
+    val resultSet = stat.executeQuery
+    val sb = new StringBuilder
+    while (resultSet.next)
+      sb.append(resultSet.getString("training_set"))
+    resultSet.close()
+    sb.toString()
+  }
+
+  def getSerialisedClassifier(exercise: String): Classifier[String, String] = {
```

Comment:
Well to keep the db updated according to feedback I do not need to interact with the marking service, I can just pull/update the training set, retrain and upload the serialized class, and by magic, the ms will use it next time it will run